### PR TITLE
Support multi dimension values and configurable metricValues on AWS C…

### PR DESCRIPTION
**Pull request by JonasMatos0**
…loudwatch Scaler

Signed-off-by: Jonas Matos <js.matos@sidi.org.br>
Signed-off-by: Thiago Teodoro Rodrigues <t.teodoro@sidi.org.br>

### CODE REVIEW

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ triggers:
   metadata:
     # Required: namespace
     namespace: AWS/SQS
-    # Required: Dimension Name
+    # Required: Dimension Name - supports multiple dimensionName with ";" i.e. dimensionName: QueueName;QueueName
```
Comment (zroubalik) [inline content/docs/2.1/scalers/aws-cloudwatch.md:20]: We should add the same for dimensionValue

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: content/docs/2.1/scalers/aws-cloudwatch.md (hunk 1 above)
Comment (JonasMatos0) [reply content/docs/2.1/scalers/aws-cloudwatch.md:20]: Updated. Can you please  take a look once again, @zroubalik ?

Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:20]:
```suggestion
    # Required: Dimension Name - Supports specifying multiple dimension names by using ";" as a seperator i.e. dimensionName: QueueName;QueueName
```

However, is it required for me to pass the same dimension name twice?

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 2]
```diff
@@ -17,8 +17,9 @@ triggers:
...
     dimensionName: QueueName
+    # Required: Dimension Value - supports multiple dimensionValue ";" i.e. dimensionValue: queue1;queue2
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:22]:
```suggestion
    # Required: Dimension Value - Supports specifying multiple dimension values by using ";" as a seperator i.e. dimensionValue: queue1;queue2
```

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 3]
```diff
@@ -30,6 +31,12 @@ triggers:
     # Optional: AWS Secret Access Key, can use TriggerAuthentication as well
     awsSecretAccessKeyFromEnv: AWS_SECRET_ACCESS_KEY # default AWS_SECRET_ACCESS_KEY
     identityOwner: pod | operator # Optional. Default: pod
+    # Optional: defaultMetricCollectionTime
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:34]: Can you please provide some docs on these new parameters in our parameter list please?

For me it's not clear what they represent.

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: content/docs/2.1/scalers/aws-cloudwatch.md (hunk 3 above)
Comment (JonasMatos0) [reply content/docs/2.1/scalers/aws-cloudwatch.md]: Updated Parameter list in the doc. Can you please review, @tomkerkhove ?

**Review by zroubalik — APPROVED** [review]
LGTM

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 4]
```diff
@@ -38,6 +45,13 @@ triggers:
 
 > When `identityOwner` set to `operator` - the only requirement is that the Keda operator has the correct IAM permissions on the CloudWatch. Additional Authentication Parameters are not required.
 
+- `defaultMetricCollectionTime` - How long in the past (seconds) should the scaler check AWS Cloudwatch. Used to define **StartTime** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:48]:
```suggestion
- `defaultMetricCollectionTime` - How long in the past (seconds) should the scaler check AWS Cloudwatch. Used to define **StartTime** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 5]
```diff
@@ -38,6 +45,13 @@ triggers:
...
+
+- `defaultMetricStat` - Which statistics metric is going to be used by the query. Used to define **Stat** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:50]:
```suggestion
- `defaultMetricStat` - Which statistics metric is going to be used by the query. Used to define **Stat** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```

File: content/docs/2.1/scalers/aws-cloudwatch.md [hunk 6]
```diff
@@ -38,6 +45,13 @@ triggers:
...
+
+- `defaultMetricStatPeriod` -  Which frequency is going to be used by the related query. Used to define **Period** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/aws-cloudwatch.md:52]:
```suggestion
- `defaultMetricStatPeriod` -  Which frequency is going to be used by the related query. Used to define **Period** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```

**Review by tomkerkhove — APPROVED** [review]
LGTM once the links are tweaked, sorry for the nit

**Pull request merged**